src/styles/main.scss

```diff
@@ -1 +1,62 @@
-// put code here
+@import "./utils/general.scss";
+@import "./utils/varibales.scss";
+
+.watch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 10px dashed #000;
+  border-radius: $watch-radius;
+  margin: 0 auto;
+  width: $wach-size;
+  height: $wach-size;
+
+  &__center-item {
+    position: absolute;
+    z-index: 10;
+    border-radius: $watch-radius;
+    background: rgb(235, 219, 8);
+    width: 40px;
+    height: 40px;
+  }
+
+  &__hour-hand {
+    position: absolute;
+    z-index: 1;
+    bottom: $hand-position;
+    height: 20vh;
+    width: 20px;
+    border-radius: $hand-radius;
+    background-color: rgb(0, 46, 253);
+    transform-origin: bottom;
+    animation-name: watch-move;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+    animation-duration: 60s;
```

Comment:
you can use here shorthand property
https://www.w3schools.com/cssref/css3_pr_animation.php